### DISCUSSION TITLE
Aluminum foam using /MAT/LAW115

### DISCUSSION THREAD

**Discussion by avocadofruit** [OpenRadioss/OpenRadioss#3228] [Q&A] [answered]
Hi all, I'm trying to simulate a blast impact of aluminum foam using the material /MAT/LAW115 (DESHFLECK). It gives me the warning: Unsupported option /MAT/LAW115 in format < 2021.

Any ideas to resolve this?

**Comment by Schwoertzig** [comment]
Hi @avocadofruit ,
If your input format is posterior to 2021, this is likely a bug. In that case, I recommend sharing an input file.
However, if your format is anterior to 2021, you’ll need to update your input file by updating the **Invers** parameter in the **/BEGIN** option. You can find more details in the official documentation here: [Radioss Manual - /BEGIN (Starter)](https://help.altair.com/hwsolvers/rad/topics/solvers/rad/begin_starter_r.htm#begin_starter_r).
Regards

**Reply by avocadofruit** [reply, marked as answer]
Hi, I managed to resolve it by changing Invers to 2024. Thanks a lot!
